sound/soc/sdw_utils/soc_sdw_utils.c

```diff
@@ -1112,8 +1114,9 @@ int asoc_sdw_parse_sdw_endpoints(struct snd_soc_card *card,
 				dai_info = &codec_info->dais[adr_end->num];
 				soc_dai = asoc_sdw_find_dailink(soc_dais, adr_end);
 
-				if (dai_info->quirk && !(dai_info->quirk & ctx->mc_quirk))
-					continue;
+				if (dai_info->quirk &&
+				    !(dai_info->quirk_exclude ^ (dai_info->quirk & ctx->mc_quirk)))
```

Comment:
Can we use `!dai_info->quirk_exclude * (dai_info->quirk & ctx->mc_quirk))`? It seems explain the `quirk_exclude` flag more.